packages/nimbus/src/components/range-calendar/range-calendar.mdx

```diff
@@ -1,93 +1,156 @@
 ---
 id: Components-RangeCalendar
 title: RangeCalendar
-description: Filler for now, not sure you're going to get a designated file. 
-documentState: InitialDraft
+description:
+  Calendar Range display a grid of days in one or more months and allow users to
+  select a range of dates.
+
 order: 999
 menu:
   - Components
   - Inputs
   - RangeCalendar
 tags:
   - component
+  - calendar
 ---
 
 # RangeCalendar
 
-Filler for now, not sure you're going to get a designated file. 
-
-## Basic usage
+Calendar Range display a grid of days in one or more months and allow users to
+select a range of dates.
 
-[Explain the basic usage / usecase of the component].
+## Overview
 
-```jsx-live
-const App = () => <RangeCalendar/>
-```
-{/* 
+The calendar component is most often used within the popover component when a
+user interacts with a date input. Easily toggle, month, year, select a range,
+and include start and end times.
```

Comment:
i believe we still need to remove the mentioning of start/end times (unless we want to mention it in a blurb with the picker?). i went to leave a note in figma, but it was already answered [here](https://www.figma.com/design/gHbAJGfcrCv7f2bgzUQgHq?node-id=3552-6340&m=dev#1432536475) 
cc @ellelevy ...oh, also adding related components to THE LIST